### PR TITLE
Add tests for skipped_fields on items and requests

### PR DESCRIPTION
**Pull request by fcanobrash**

### CODE REVIEW

**Comment by fcanobrash** [comment]
The tests are failing because the changes depend on PR #42 .

File: tests/test_record.py [hunk 1]
```diff
@@ -185,3 +199,34 @@ def test_dotted_name(self):
...
+
+    def test_skipped_fields(self):
+        with CaseSpider() as spider:
+            spider.imports('import time')
+            spider.custom_settings('''
+                AUTOUNIT_SKIPPED_FIELDS = ['ts']
+            ''')
+            spider.start_requests("yield scrapy.Request('data:text/plain,')")
+            spider.parse('''
+                yield {'a': 4, 'ts': time.time()}
```
Comment (andrewbaxter) [inline tests/test_record.py:212]: As unlikely as it is, it's possible this test could pass despite being broken if `spider.test()` runs before 1ms has elapsed...

I can't think of a workaround that wouldn't be really ugly though.  Maybe add a very very short sleep here?

Comment (fcanobrash) [reply tests/test_record.py:212]: Oh right. Sure, we can add a small sleep here to prevent that.

File: tests/test_record.py [hunk 2]
```diff
@@ -185,3 +199,35 @@ def test_dotted_name(self):
...
+
+    def test_skipped_fields(self):
+        with CaseSpider() as spider:
+            spider.imports('import time')
+            spider.custom_settings('''
+                AUTOUNIT_SKIPPED_FIELDS = ['ts']
+            ''')
+            spider.start_requests("yield scrapy.Request('data:text/plain,')")
+            spider.parse('''
+                time.sleep(0.5)
```
Comment (andrewbaxter) [inline tests/test_record.py:211]: I think you can just sleep between `spider.record()` and `spider.test()` but this works too :D

**Review by andrewbaxter — APPROVED** [review]

**Pull request merged**